docs/DeveloperGuide.md

```diff
@@ -329,24 +329,27 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 <hr>
 
-**Use case: Delete a TA**
+**Use case: Delete a TA from the system**
 
 **MSS**
 
 1.  User requests to list TA applicants
-2.  TAlent Assistant™ shows the list of TA applicants
-3.  User requests to delete an existing TA from the list
+2.  TAlent Assistant™ displays the list of TA applicants
+3.  User requests to delete a specific TA in the list
 4.  TAlent Assistant™ deletes the TA from the list and displays the deleted entry
 
     Use case ends.
 
 **Extensions**
 
-* 3a. The given reference, i.e. Student Number, is invalid.
+* 3a. The list is empty.
+    
+    Use case ends
+* 3b. The given student number is invalid or does not exist in the system
 
-    * 3a1. TAlent Assistant™ shows an error message.
+    * 3a1. TAlent Assistant™ displays an error message
 
-  Use case resumes at step 2.
+  Use case resumes at Step 2.
```

Comment:
Perhaps `Step` should remain as lower case? 🤔

Reply:
Thanks for spotting the mistake! Fixed.

Reply:
👍